elytron/src/main/java/org/wildfly/extension/elytron/FileSystemRealmDefinition.java

```diff
@@ -157,10 +199,27 @@ protected void performRuntime(OperationContext context, ModelNode operation, Mod
 
             final String hashEncoding = HASH_ENCODING.resolveModelAttribute(context, model).asString();
             final String hashCharset = HASH_CHARSET.resolveModelAttribute(context, model).asString();
+            final String credentialStore = CREDENTIAL_STORE.resolveModelAttribute(context, model).asStringOrNull();
+            final String secretKey = SECRET_KEY.resolveModelAttribute(context, model).asStringOrNull();
+
+
+            String credentialStoreRuntimeCapability;
+            ServiceName credentialStoreServiceName = null;
+            if(credentialStore != null) {
+                credentialStoreRuntimeCapability = buildDynamicCapabilityName(CREDENTIAL_STORE_CAPABILITY, credentialStore);
+                credentialStoreServiceName = context.getCapabilityServiceName(credentialStoreRuntimeCapability, CredentialStore.class);
```

Comment:
@Ashpan Just minor, credentialStoreServiceName is unused and this block can be removed since the dependency on credential store service is defined below.

Reply:
Removed